packages/magmo-wallet-client/src/wallet-events.ts

```diff
@@ -241,26 +241,48 @@ export const hideWallet = () => ({
  */
 export type HideWallet = ReturnType<typeof hideWallet>;
 
-// MESSAGING
+// WALLET-TO-WALLET COMMUNICATION
 // =========
 /**
- * The type of event when a message to the opponent's wallet is requested.
+ * The type of event when a message relay to the opponent's wallet is requested.
  */
-export const MESSAGE_REQUEST = 'WALLET.MESSAGING.MESSAGE_REQUEST';
+export const MESSAGE_RELAY_REQUESTED = 'WALLET.MESSAGING.MESSAGE_REQUEST';
 /**
  * @ignore
  */
-export const messageRequest = (to: string, data: Commitment | string, signature: string) => ({
-  type: MESSAGE_REQUEST as typeof MESSAGE_REQUEST,
+export const messageRelayRequested = (to: string, data: string) => ({
+  type: MESSAGE_RELAY_REQUESTED as typeof MESSAGE_RELAY_REQUESTED,
   to,
   data,
+});
+
+/**
+ * The event thrown when the wallet requests a message be relayed to the opponent's wallet.
```

Comment:
Not sure about the "thrown" language here. Typically, "throw" is used for errors and exceptions.